packages/browser/src/browser/index.ts

```diff
@@ -83,9 +83,16 @@ export function loadLegacySettings(
   const baseUrl = cdnURL ?? getCDN()
 
   return fetch(`${baseUrl}/v1/projects/${writeKey}/settings`)
-    .then((res) => res.json())
+    .then((res) => {
+      if (!res.ok) {
+        return res.text().then((statusText) => {
```

Comment:
Hey, maybe I'm missing something, but why do we need text() here?


```
void fetch('https://httpstat.us/300')
  .then((res) => {
    if (!res.ok) {
      throw res
    }
  })
  .catch((e) => {
    console.error(e.statusText)
    throw err
 })
```

Reply:
Just throwing didn't give us the status text iirc, but I'll look again very quickly.

Reply:
Works in my REPL
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/5115498/183752246-58b22ceb-b8bf-4811-9c17-ff485567fd86.png">

Reply:
Confirmed, needed to unwrap the `text` in order to get the statusText.

Reply:
hmmm... `statusText` is part of the [response specification](https://developer.mozilla.org/en-US/docs/Web/API/Response/statusText)